relayer/src/chain/counterparty.rs

```diff
@@ -425,26 +426,18 @@ pub fn unreceived_acknowledgements_sequences(
 pub fn unreceived_packets(
     chain: &impl ChainHandle,
     counterparty_chain: &impl ChainHandle,
-    channel: &IdentifiedChannelEnd,
-) -> Result<Vec<u64>, Error> {
-    let counterparty = channel.channel_end.counterparty();
-    let counterparty_channel_id = counterparty
-        .channel_id
-        .as_ref()
-        .ok_or_else(Error::missing_counterparty_channel_id)?;
-
-    let (commit_sequences, _) = commitments_on_chain(
+    path: &PathIdentifiers,
+) -> Result<(Vec<u64>, Height), Error> {
+    let (commit_sequences, h) = commitments_on_chain(
         counterparty_chain,
-        &counterparty.port_id,
-        counterparty_channel_id,
+        &path.counterparty_port_id,
+        &path.counterparty_channel_id,
     )?;
 
-    unreceived_packets_sequences(
-        chain,
-        &channel.port_id,
-        &channel.channel_id,
-        commit_sequences,
-    )
+    let sn =
```

Comment:
Can we make this a more explicit and descriptive variable name?

Reply:
64a4531e